lib/broadside/target.rb

```diff
@@ -73,5 +75,15 @@ def env_vars
         end
       end.map { |k, v| { 'name' => k, 'value' => v } }
     end
+
+    def to_h
+      {
+        Target: @name,
```

Comment:
oh i see you're using it to build the `list_targets` stuff.  nevermind most of what i said then, though let's go with `to_hash`.